tests/test_clos/test_process_workflow_list.py

```diff
@@ -0,0 +1,37 @@
+"""pytest for fucntion process_workflow_list"""
```

Comment:
```suggestion
"""pytest for method Cloudos.process_workflow_list"""
```
Although it's not really necessary to add this docstring to this simple file.